### PR TITLE
Add pointer to individual extension documentation

### LINKED ISSUE #593: Document the extensions that we provide
We need to add documentation for the extensions that we provide. These can be documented directly in the repository, with a link in our extensions docs on readme.com

## Acceptance Criteria

- [x] Add a `README.md` file to the top-level `extensions` folder that contains a list of our extensions with a brief description of each
- [x] Add a `README.md` to each extension folder that explains the extension in greater detail including examples of how to run the extension and any supported arguments, etc.
- [x] Add a section to `docs/extensions/extension_overview.md` that will tell users that we have written a small set of extensions and include a link to the folder on GitHub for more information.

### PR DESCRIPTION
**Pull request by cd-work**
This adds some pointers at our documentation root levels, pointing users
to the individual official Phylum extensions which they can try out.

Closes #593.

### CODE REVIEW

File: extensions/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Phylum Extensions
+
+This directory contains all official Phylum CLI extensions. To get started just
+pick the extension you want to install and follow the instruction in their
```
Comment (maxrake) [inline extensions/README.md:4]:
```suggestion
pick the extension you want to install and follow the instructions in their
```

File: extensions/duplicates/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Phylum duplicates extension
+
+A [Phylum CLI][phylum-cli] extension that checks for duplicates in your
```
Comment (maxrake) [inline extensions/duplicates/README.md:3]: This is a missing link.

**Review by maxrake — CHANGES_REQUESTED** [review]
One of the acceptance criteria for #593 is:

> Add a README.md file to the top-level extensions folder that contains a list of our extensions with a brief description of each

The `README.md` was added but the list and description of each extension was not. Was that on purpose?

**Review by andreaphylum — DISMISSED** [review]

**Comment by cd-work** [comment]
> The README.md was added but the list and description of each extension was not. Was that on purpose?

Yes, I disagree with the suggestion in the original issue to describe each extension inside the root directory. This adds pointless maintenance having to describe each extension in the readme for the extension and in the root. Keeping it light in the root makes things a lot simpler and we can grow that directory fearlessly without having to constantly update the same file.

I'm confident people are capable of selecting an extension based on its name and then read that specific readme to get more information about what it does.

**Review by maxrake — APPROVED** [review]

**Pull request merged**